src/hoc/asPaginationItem/index.js

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import PropTypes from "prop-types";
+import glamorous from "glamorous";
+import { BLUE, GREY_4, PURPLE, YELLOW, WHITE } from "govuk-colours";
+import {
+  FONT_SIZE,
+  LINE_HEIGHT,
+  SPACING,
+  MEDIA_QUERIES,
+  NTA_LIGHT
+} from "../../constants/index";
+
+import NextPageIcon from "../../icons/ArrowRight/index";
+import PrevPageIcon from "../../icons/ArrowLeft/index";
+
+const PaginationWrapper = glamorous.li(
+  {
+    boxSizing: "border-box",
+    margin: 0,
+    padding: 0,
+    listStyleType: "none",
+    fontFamily: NTA_LIGHT,
+    fontWeight: 400,
+    textTransform: "none",
+    fontSize: FONT_SIZE.SIZE_20,
+    lineHeight: LINE_HEIGHT.SIZE_16,
+    width: "100%",
+    [MEDIA_QUERIES.LARGESCREEN]: {
+      fontSize: FONT_SIZE.SIZE_27,
+      lineHeight: LINE_HEIGHT.SIZE_16
+    },
+    "> a": {
+      boxSizing: "border-box",
+      color: BLUE,
+      fill: "currentColor",
+      padding: SPACING.SCALE_3,
+      backgroundColor: WHITE,
+      textDecoration: "none",
+      outlineColor: "transparent",
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "center",
+      width: "100%",
+      ":hover": {
+        backgroundColor: GREY_4
+      },
+      ":focus": {
+        outline: `3px solid ${YELLOW}`
+      },
+      ":visited": {
+        color: PURPLE
+      }
+    }
+  },
+  ({ previousPage }) => ({
+    marginRight: previousPage ? "3px" : undefined,
+    " > a": {
+      alignItems: previousPage ? "flex-start" : undefined
+    },
+    " > a div": {
+      justifyContent: previousPage ? "flex-start" : undefined,
+      " > svg": {
+        marginRight: previousPage ? "10px" : undefined
+      }
+    }
+  }),
+  ({ nextPage }) => ({
+    " > a": {
+      alignItems: nextPage ? "flex-end" : undefined
+    },
+    " > a div": {
+      justifyContent: nextPage ? "flex-end" : undefined,
+      " > svg": {
+        marginLeft: nextPage ? "10px" : undefined
+      }
+    }
+  })
+);
+
+const InnerWrap = glamorous.div({
+  display: "flex",
+  alignItems: "center",
+  width: "100%"
+});
+
+const PageTitle = glamorous.span({
+  fontSize: FONT_SIZE.SIZE_14,
+  lineHeight: LINE_HEIGHT.SIZE_14,
+  textDecoration: "underline",
+  [MEDIA_QUERIES.LARGESCREEN]: {
+    fontSize: FONT_SIZE.SIZE_16,
+    lineHeight: LINE_HEIGHT.SIZE_16
+  }
+});
+
+const asPaginationItem = AnchorType => {
+  const component = props => (
+    <PaginationWrapper
```

Comment:
Do we have to use a span wrapper? e.g. with `glamorous(AnchorType)`

I think we discussed this for the anchor hoc but not sure if we resolved it.

Maybe we should add a new issue to look in to this?

Reply:
okay. I spent a bit of time last week trying to style the hoc but didnt have much luck. i felt that wrapping a glamorous tag around was a quick and didnt seem to have many issues (apart from an extra tag)